config/groovy/common.groovy

```diff
@@ -182,12 +182,47 @@ class common {
         addRemote(itemName, defaultRemote, "https://github.com/${githubDefaultHome}/${itemName}.git")
     }
 
+    /**
+     * Check if an item was updated within the provided time limit
+     * @param fetchHead the item's FETCH_HEAD file in the .git directory
```

Comment:
This methods works independently of the fact that you pass it a `FETCH_HEAD` file. Thus, I would just call this `file` to make that clear. 

Using this method to determine whether the git state is somewhat recent is a usage specific.

Reply:
Good point.